docs/how_to/linking_sorted_data.rst

```diff
@@ -0,0 +1,149 @@
+.. _linking_sorted_data:
+
+How to Link Sorted Data to Electrodes
+=====================================
+
+When converting spike sorting results to NWB format, it is essential to preserve the
+relationship between sorted units and the recording electrodes that detected them.
+This linkage ensures that each unit inherits all electrode-level metadata stored in the
+`electrodes` table of the NWB file.
+
+For this linkage to be useful, the `electrodes` table itself must be **well annotated**,
+including accurate information on brain area, anatomical coordinates, electrode geometry,
+and any probe-specific metadata. Without this detail, the benefits of unit-electrode
+linking are severely limited. For guidance on electrode table annotation, see
+:doc:`annotate_ecephys_data`.
+
+Why Link Units to Electrodes?
+-----------------------------
+
+Proper electrode linking allows each unit to be formally connected to all the metadata
+describing its recording site. This enables both spatial and anatomical localization
+of units which is critical for accurate interpretation and reproducibility:
+
+**Spatial Analysis**
+    With well-annotated electrode positions (e.g., rel_x, rel_y, rel_z),
+    future users of the NWBFile can determine where units lie within the probe, perform laminar
+    analyses, assess depth-dependent firing properties, and investigate spatial
+    organization such as receptive field gradients or clustering patterns across channels.
+
+**Anatomical Analysis**
+    Registering the probe's position in the brain allows anatomical features such as
+    brain area, subregion, or cortical layer to be associated with electrodes and,
+    by extension, with linked units.
+    As an example, `Liu et al. (2022) <https://doi.org/10.1038/s41467-022-33536-x>`_ demonstrated how depth-resolved recordings across
+    hippocampal layers reveal distinct current source density and local field potential
+    signatures of sharp wave-ripples. This type of interpretation is only possible when
+    recording channel locations are known and correctly linked to sorted units.
+
+**Quality Control and Traceability**
+    Linking units to electrode metadata ensures full traceability from spike sorting
+    results back to the raw recording channels. This allows you to inspect waveforms,
+    review spike detection events, and confirm that units are spatially plausible
+    (e.g., waveforms localized to nearby electrodes). Such verification helps detect
+    sorting errors, identify artifacts, and maintain reproducibility by making the
+    sorting process transparent and auditable.
+
+
+Accessing Electrode Metadata from Units
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Once units are properly linked to electrodes and the electrodes table is well annotated,
+you can programmatically retrieve electrode-level metadata for any unit in your NWB file.
+
```

Comment:
I guess this is a personal preference, but I would change the order to first show how to link units to electrodes and then how to access them

Reply:
Thanks for the review, I am gonna invert the order in a separate PR after https://github.com/catalystneuro/neuroconv/pull/1449 to avoid merge conflicts.